src/main/java/org/humancellatlas/ingest/security/authn/provider/gcp/GoogleServiceJwtAuthenticationProvider.java

```diff
@@ -47,7 +49,9 @@ public Authentication authenticate(Authentication authentication) throws Authent
             Authentication jwtAuth = DelegatingJwtAuthentication.delegate(jwt, jwtVerifier);
```

Comment:
Are we using that variable at all: `jwtAuth`?

Reply:
Yup, it's being used in the logs. We definitely need to call the `DelegatingJwtAuthentication.delegate(jwt, jwtVerifier)` because token verification happens there